.github/workflows/integration-tests.yml

```diff
@@ -0,0 +1,67 @@
+name: Integration tests
+
+on:
+  workflow_dispatch:
```

Comment:
This shouldn't be workflow dispatch. It must be pull_request to test on CI

Reply:
then it should be part of the `ci.yml`? I can move it over; I was just copying from jcloud's setup, where we need to trigger manually / by schedule, not depending on PR.

Reply:
Yah. This has to be a part of CI

Reply:
Ok, should be fixed by now.